### DISCUSSION TITLE
Hide admin page

### DISCUSSION THREAD

**Discussion by JoseEspadas** [AzuraCast/AzuraCast#6282] [General]
Hello good,
I am currently using Azuracast and my question is, .
How can I hide the administration page, so that no one can see the radios, the listeners that are currently listening, etc.

Thank you so much..

**Comment by JoseEspadas** [comment]
How to hide this page and nobody can see it?
https://radios.lamaster.es:8000/

**Reply by gunsar** [reply]
maybe it can be custom from icecast using an alias or redirect to the mountpoint stream url

**Reply by unudelaradio** [reply]
To make it look like that, go to sidebar > Broadcasting > Mount Points and uncheck _Show on Public Pages_ for your mountpoint.

**Reply by mdrxy** [reply]
> To make it look like that, go to sidebar > Broadcasting > Mount Points and uncheck _Show on Public Pages_ for your mountpoint.

Do we know is this alters visibility in YP listings? I've just toggled the setting on my station, though I know it can take some time for updates to populate to third parties.
